### ISSUE TITLE
Quantization-aware training: quantized parameters are still floating-point numbers

### ISSUE THREAD

**Issue by lehahoang** [pytorch/tutorials#1044]
Hi,

Just for curiosity, I examined the quantized parameters before being evaluated by printing them out, and here are the results.
> ....
>  [[-3.5608e-03, -5.9347e-03, -2.0475e-02],
>           [-8.0119e-03,  3.8576e-03,  9.7923e-03],
>           [-1.1869e-03,  6.8249e-03,  1.7211e-02]]]], size=(32, 3, 3, 3),
>        dtype=torch.qint8, quantization_scheme=torch.per_channel_affine,
>        scale=tensor([4.1970e-05, 2.5147e-03, 2.4814e-03, 2.7990e-03, 2.8973e-04, 5.4064e-04,
>         1.0432e-03, 4.7924e-04, 7.9272e-04, 8.2421e-06, 1.1249e-03, 2.1563e-04,
>         2.3528e-03, 1.0647e-03, 1.2701e-05, 1.1407e-05, 3.4314e-05, 5.2139e-06,
>         1.5107e-04, 9.6313e-04, 7.7032e-04, 7.1193e-04, 1.8537e-03, 6.7458e-06,
>         8.8377e-04, 1.4991e-03, 6.6087e-04, 1.5101e-03, 5.4526e-04, 1.0136e-03,
>         1.4105e-03, 2.9674e-04], dtype=torch.float64),
>        zero_point=tensor([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
>         0, 0, 0, 0, 0, 0, 0, 0]),
>        axis=0)

It seems like they are still floating-point numbers after the quantization step. I expected that they were 8-bit unsigned Integer values, but it turned out to be not. As a result, do you guys have any idea about this? I just simply add the print syntax as the following.
> ...
>         quantized_model = torch.quantization.convert(qat_model.eval(), inplace=False)
>         quantized_model.eval()
>         print(quantized_model.state_dict()['features.0.0.weight'])
>         top1, top5 = evaluate(quantized_model,criterion, data_loader_test, neval_batches=num_eval_batches)
>         print('Epoch %d :Evaluation accuracy on %d images, %2.2f'%(nepoch, num_eval_batches * eval_batch_size, top1.avg))

Thank you.
BR

**Comment by huangsx95** [comment]
get the underlying integer representation of the quantized Tensor by `int_repr()`.
e.g. `print(quantized_model.state_dict()['features.0.0.weight'].int_repr())`
